### PR TITLE
Charthub Revamp

### PR DESCRIPTION
**Pull request by gdsoumya**

### CODE REVIEW

File: app/client/src/components/Charts/index.tsx [hunk 1]
```diff
@@ -19,38 +16,33 @@ const getIconUrl = (chartMetadataName: string, chartGroup: string) => {
 	) {
 		baseURL = `${window.location.protocol}//${window.location.hostname}:8080`;
 	} else baseURL = "/api";
+	if (chartMetadataName === "all-experiments")
```
Comment (chandankumar4) [inline app/client/src/components/Charts/index.tsx:19]: why are we filtering the chart based on name?

File: app/client/src/components/CustomCard/CardContainer.tsx [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
+import clsx from "clsx";
 
 function CardContainer(props: any) {
 	const classes = useStyles();
-	return <div className={classes.card}>{props.children}</div>;
+	return (
+		<div
+			key={props.key}
+			className={clsx(
+				props.children.props.title !== "all-experiments"
```
Comment (chandankumar4) [inline app/client/src/components/CustomCard/CardContainer.tsx:11]: same comment ad above

File: app/client/src/components/Charts/index.tsx (hunk 1 above)
Comment (gdsoumya) [reply app/client/src/components/Charts/index.tsx:19]: This is required, when the user goes for all-experiments we do not show an experiment page rather the page which has the link to install all the experiments. To do this we need to check the experiment name.

File: app/client/src/components/CustomCard/CardContainer.tsx (hunk 2 above)
Comment (gdsoumya) [reply app/client/src/components/CustomCard/CardContainer.tsx:11]: Same as above

File: app/client/src/components/Charts/index.tsx (hunk 1 above)
Comment (gdsoumya) [reply app/client/src/components/Charts/index.tsx:19]: It also has a different design so we have to identify it

**Comment by chandankumar4** [comment]
Check the build failure issue once!!

**Comment by gdsoumya** [comment]
@chandankumar4 Build has passed

**Comment by chandankumar4** [comment]
> @chandankumar4 Build has passed

instead of removing the Cypress test can we comment that, so that we can modify and use that later

**Comment by gdsoumya** [comment]
> instead of removing the Cypress test can we comment that, so that we can modify and use that later

We would have to write the tests from scratch as it is because the whole website has changed.

**Pull request merged**